### PR TITLE
official_devices: Add Cheeseburger and Dumpling

### PR DESCRIPTION
**Pull request by karthik990**

### CODE REVIEW

File: devices.json [hunk 1]
```diff
@@ -158,5 +158,25 @@
            "has_vanilla_builds": false,
            "has_gapps_builds": false
         }
-    }    
+    };
```
Comment (mohancm) [inline devices.json:161]: fix this

File: devices.json [hunk 2]
```diff
@@ -158,5 +158,25 @@
...
+    {
+       "name": "Oneplus 5",
+       "brand": "Oneplus",
+       "codename": "cheeseburger",
+       "xda_thread": "https://forum.xda-developers.com/t/rom-11-0-dotos-5-0-oneplus-5-t-official-22-mar-2021.4233475/",
+       "version_info": {
+           "has_vanilla_builds": false,
+           "has_gapps_builds": false
+        }
+    };
```
Comment (mohancm) [inline devices.json:171]: it should be `comma`, not semicolon

**Review by mohancm — CHANGES_REQUESTED** [review]
JSON malformed

File: devices.json (hunk 2 above)
Comment (karthik990) [reply devices.json:171]: Damn my bad

**Pull request closed without merging**